Entity: Case

```diff
@@ -0,0 +1,74 @@
+User account that registered the case in SORMAS
+6.	Reporting user
+User account that registered the case in SORMAS
+7.	Classifying user
+User account that classified the case from “not yet classified” to any other classification
+8.	Case classification
+Classification assigned to a case
+9.	Date of classification
+Date a case was classified to any other category apart from the default “Not yet classified”
+10.	Investigation status
+Information on the investigation status (the activity of the public health officer): pending/done/discarded investigation
+11.	Investigation date
+Start date of an investigation
+12.	Disease
+Type of disease (working diagnosis) of a case
+13.	Outcome of case
+Status of the case at the end of the clinical course: recovered or deceased.
+14.	Case origin
+Place where a case was identified. Either port of entry (ground crossing or air port or sea port) or in-country (e.g. hospital, home, school, care home)
+15.	Region
+Region in the country where a case was identified (this might not be the region of residence of the person as documented/captured under “case person”).
+16.	District
+District within a region where a case was identified (this might not be the district of residence of the person as documented/captured under “case person”).
+17.	Community
+Community within a region where the case was identified (this might not be the community of residence of the person as documented/captured under “case person”).
+18.	Health facility
+Health facility within the community where a case was identified. Where available, select name of medical care facility from the drop-down menu
+19.	Report GPS latitude/longitude/accuracy in metres
+Place where the case was reported. Using a mobile device, the fields are filled automatically or it can be entered manually with information from any GPS device
+20.	Quarantine (place of quarantine)
+A place (home, designated facility, etc.) where persons who were exposed to a contagious disease are kept separated from the general population under restricted movement to see if they become sick within the incubation period of the disease in question). 
+21.	Name of responsible surveillance officer
+A public health officer at the district level who has been assigned the responsibility/task of following-up on a case.
+22.	Name of responsible clinician
+The name of a clinician in charge of a case
+23.	General comment
+Any additional information or remark related to the case.
+24.	Archive
+Refers to old cases that are not necessary for the daily work (e.g. recovered cases) that have been stored away from the user interface of the system. Thus, archived cases are not loaded by default to the user interface of the system, and this is important in order not to slow down system’s processing speed. Should the user wants to see archived cases in the case directory, a click on the case directory will open a pull-down menu next to “active cases”. The may now select “archived cases” or “all cases”.
+25.	Delete
```

Comment:
This is a description of how certain buttons in the UI work, rather than a description of the database entity if I'm not mistaken?